.golangci.reference.yml

```diff
@@ -1847,6 +1847,11 @@ linters-settings:
     # Default: true
     strict-append: true
 
+  contextcheck:
+    # If you use contextcheck in docker ci, and feel a little bit slow, you can turn on this option.
```

Comment:
If you have the same report with or without facts, why do you need it?

The option explanation is not clear what this option does.

Reply:
golang-ci default have cache feature, this can save last issus/fact if source not change.

but, contextcheck need cross over depend packages, if not use Fact, cached depend package not run any pass.

but open Fact, any depend packages(more than close Fact) will run SSA pass, so need more time for it..

can give same advice for this comment... 😊

Reply:
in summary: if the facts are disabled the linter will produce false positives but it will be faster.
So the option is really questionable: do we need an option to produce false positives?

Reply:
removed config, clear the doubt 😊